### ISSUE TITLE
Right-clicking breaks PCem window in libTAS

### ISSUE THREAD

**Issue by slamotas** [TASEmulators/pcem-old#2]
I was testing TASing with the mouse in PCem and I came across an issue. If you hold the right mouse button and then frame advance, the menu for floppy/CD mounting pops up and the PCem window becomes unresponsive (you can still frame advance but it will no longer accept inputs). This even happens if you record a right mouse button click in the input editor and play the movie back.

I'm not sure about the best way to handle this. I guess disable the pop-up menu? We would have to keep an unpatched version of PCem on hand to mount floppies/CD into a configuration, or just edit the config file manually.

**Comment by clementgallet** [comment]
Yeah, I expected right-click menu to be buggy, and there's nothing easy I can do on libTAS's side to fix it. Users would need to specify floppies/cd paths in the config file. Also, it means we don't support hotswapping disks.

I'm not sure we need to specifically disable right click, if we want to run the emulator natively as well, for testing purpose.

**Comment by vadosnaprimer** [comment]
Can't the problematic commands attached to right-click be reattached to something else?

**Comment by clementgallet** [comment]
If needed, yes, we could implement shortcuts for specific features we need.
